dali/base/dautils.cpp

```diff
@@ -2109,7 +2138,10 @@ IRemoteConnection *getElementsPaged( IElementsPager *elementsPager,
     {
         elem.setown(QUERYINTERFACE(pagedElementsCache->get(owner,*hint),CPECacheElem)); // NB: removes from cache in process, added back at end
         if (elem)
+        {
             postfilter = elem->postFilter; // reuse cached postfilter
+            elem->resetDue();
```

Comment:
This is in effect a separate unrelated bug fix isn't it?
i.e. reused existing pagers were not being updated and would have been considered old, even though in use..

For clarity (of later looking at this bug/symptom) it would be better if in a separate JIRA/PR if possible..
if the code becomes too intertwined when you're updating to fix the primary issue then it's okay, but need to update the JIRA and PR commit messages to say that this secondary issues was fixed as well.

Reply:
It would make more sense if add() set the due time and the CTimedCacheItem constructor did not (left it set to 0).

Reply:
Yes, you are correct. The reused existing pagers should still be considered old. If it should be re-considered later, we may create a separate JIRA.